scalameta/tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -0,0 +1,26 @@
+scalahost/integration/src/main/scala/example/Example.scala
+----------------------------------------------------------
+Dialect:
+Scala212
```

Comment:
That's fine for me by now. We can normalize the dialect to something else if you prefer. You won't have to fix this file manually though, just run the SaveExpect... main and commit.